contracts/number/types/Fixed6.sol

```diff
@@ -82,6 +82,16 @@ library Fixed6Lib {
         return roundOut ? Fixed6.wrap(NumberMath.divOut(Fixed18.unwrap(a), 1e12)): from(a);
     }
 
+    /**
+     * @notice Creates a signed fixed-decimal from a significand and an exponent
+     * @param significand Significand
+     * @param exponent Exponent
+     * @return New signed fixed-decimal
+     */
+    function fromSignificandAndExponent(int256 significand, uint256 exponent) internal pure returns (Fixed6) {
```

Comment:
let's stick to the pattern of using `from` for all overloads of the from function.

(we still want the custom name in the mocks to for disambiguation).